### PR TITLE
Place staff skills in slider in mobile view fixes #1950

### LINKED ISSUE #1950: Place staff skills in slider in mobile view
**Actual Behaviour**
In mobile view the staff skills are shown one below the other.

**Expected Behaviour**
Place staff skills in slider in mobile view

**Steps to reproduce it**

**LogCat for the issue**

**Screenshots of the issue**
Screenshot 2019-04-04 at 11 59 10 AM

**Would you like to work on the issue?**
I am working on it.

### PR DESCRIPTION
**Pull request by kshitijk83**
Fixes #1950 

Changes: 1. removing the styling through DOM object for staffPicks section
2. invoking function scrollRight and scrollLeft in resize event listener

Surge Deployment Link: https://pr-1951-fossasia-susi-skill-cms.surge.sh

Screenshots for the change: 
Screenshot from 2019-04-07 15-35-13

#1950

### CODE REVIEW

**Comment by kshitijk83** [comment]
please don't use chrome developer tool for seeing in mobile device, but to see please reload the browser after resizing the window

**Comment by kshitijk83** [comment]
@AakashMallik @harsh-jindal

**Comment by harsh-jindal** [comment]
@kshitijk83 Hide the left button for staff skills in web view.
And make the list scrollable to the right instead of left when the page is loaded.
Screenshot 2019-04-07 at 7 06 53 PM

**Comment by kshitijk83** [comment]
ok i am working on it

**Comment by kshitijk83** [comment]
@harsh-jindal @AakashMallik , i resolved this, you can check it now

**Comment by AnupKumarPanwar** [comment]
Good work. I would suggest that we can remove those slider arrows on mobile view as swiping the cards is more convenient.

**Comment by kshitijk83** [comment]
@AnupKumarPanwar, I think removing those sliders is not for good UX as sometimes the person won't able to find if the sections are scrollable or not, but if it helps i can do that, but if others will agree, you can create another issue for that.

File: src/components/SkillCardScrollList/SkillCard.js [hunk 1]
```diff
@@ -60,36 +51,37 @@ class SkillCard extends Component {
         break;
       default:
         scrollCards = 1;
+        this.setState({ rightBtnDisplay: 'inline' });
     }
-    let width = window.innerWidth - 317;
+    // if(this.state.leftBtnDisplay==='none'&&this.state.rightBtnDisplay==='none'){
```
Comment (harsh-jindal) [inline src/components/SkillCardScrollList/SkillCard.js]: Remove comments. As you are not using this code.

Comment (kshitijk83) [reply src/components/SkillCardScrollList/SkillCard.js]: ok sir

**Comment by kshitijk83** [comment]
@AdityaSrivast  @harsh-jindal i have removed these comments

**Comment by kshitijk83** [comment]
@fragm3 done sir

File: src/components/SkillCardScrollList/SkillCard.js [hunk 2]
```diff
@@ -239,10 +227,7 @@ class SkillCard extends Component {
   render() {
     let leftFabStyle = styles.leftFab;
     let rightFabStyle = styles.rightFab;
-    if (window.innerWidth < 430) {
-      leftFabStyle.display = 'none';
-      rightFabStyle.display = 'none';
-    }
+    let gridlist = styles.gridList;
```
Comment (fragm3) [inline src/components/SkillCardScrollList/SkillCard.js]: Use Object destructuring

Comment (fragm3) [inline src/components/SkillCardScrollList/SkillCard.js]: Use Object destructuring

File: src/components/SkillCardScrollList/SkillCard.js [hunk 3]
```diff
@@ -60,36 +51,33 @@ class SkillCard extends Component {
...
+        : this.setState({ leftBtnDisplay: 'inline' });
+    }
   };
 
   scrollLeft = () => {
     let parentEle = document.getElementById(this.props.scrollSkills);
     let maxScrollValue = parentEle.scrollWidth - parentEle.clientWidth;
     let scrollValue = parentEle.scrollLeft - 280 * this.state.scrollCards;
     scrollAnimation(parentEle, scrollValue, 100, 'horizontal');
+    console.log(maxScrollValue, parentEle.scrollWidth, parentEle.clientWidth);
```
Comment (fragm3) [inline src/components/SkillCardScrollList/SkillCard.js]: Remove console.log

**Comment by kshitijk83** [comment]
@fragm3 it's done

File: src/components/SkillCardScrollList/SkillCard.js [hunk 4]
```diff
@@ -28,18 +28,9 @@ class SkillCard extends Component {
...
-
-    let element = document.getElementsByClassName('scrolling-wrapper')[0];
-
-    if (window.innerWidth >= 430) {
-      element.style = { width };
-    } else {
-      element.style.width = window.innerWidth;
-      element.style.margin = '10px 0px 10px 0px';
-    }
+    window.addEventListener('resize', () => {
```
Comment (akshatnitd) [inline src/components/SkillCardScrollList/SkillCard.js]: You need to remove this event listener in component unmount.

**Comment by kshitijk83** [comment]
@akshatnitd please see the changes

**Comment by kshitijk83** [comment]
@akshatnitd i think now the requested changes have been made, you can see it.

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by kshitijk83** [comment]
@mariobehling done.

**Pull request merged**